{{cookiecutter.project_slug}}/clients/web/react/package.json

```diff
@@ -15,50 +15,51 @@
   },
   "dependencies": {
     {% if cookiecutter.use_graphql == 'n' -%}
-    "@tanstack/react-query": "^4.24.2",
-    "axios": "^1.2.6",
+    "@tanstack/react-query": "^4.33.0",
+    "axios": "^1.5.0",
     "@thinknimble/tn-models": "^2.0.0",
     {% else -%}
     "@apollo/client": "^3.4.17",
     "graphql": "16.6.0",
     "graphql-tag": "^2.12.6",
     {% endif -%}
-    "@thinknimble/tn-forms": "^3.0.0",
-    "@thinknimble/tn-forms-react": "^1.0.0",
+    "@thinknimble/tn-forms": "^3.1.8",
+    "@thinknimble/tn-forms-react": "^1.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-icons": "^3.0.0",
-    "react-router-dom": "^6.0.1",
+    "react-icons": "^4.10.1",
+    "react-router-dom": "^6.15.0",
+    "zod": "3.22.2",
     "zustand": "^4.4.0"
   },
   "devDependencies": {
-    "@tanstack/eslint-plugin-query": "^4.29.9",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
+    "@tanstack/eslint-plugin-query": "^4.34.1",
+    "@testing-library/jest-dom": "^6.1.2",
+    "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.4.3",
-    "@types/react": "^18.0.24",
-    "@types/react-dom": "^18.0.8",
-    "@types/react-router-dom": "^5.3.2",
-    "@typescript-eslint/eslint-plugin": "^5.45.0",
-    "@typescript-eslint/parser": "^5.45.0",
-    "@vitejs/plugin-react": "^2.2.0",
-    "autoprefixer": "^10.4.13",
-    "cypress": "^8.6.0",
-    "eslint": "^8.28.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react": "^7.31.11",
+    "@types/react": "^18.2.21",
+    "@types/react-dom": "^18.2.7",
+    "@types/react-router-dom": "^5.3.3",
+    "@typescript-eslint/eslint-plugin": "^6.4.1",
+    "@typescript-eslint/parser": "^6.4.1",
+    "@vitejs/plugin-react": "^4.0.4",
+    "autoprefixer": "^10.4.15",
+    "cypress": "^12.17.4",
```

Comment:
this update is failing the tests, let's revert for now and **create an issue ** to tackle it separately